pkg/commoncfg/mysql_test.go

```diff
@@ -0,0 +1,80 @@
+package commoncfg
+
+import (
+	"strconv"
+	"testing"
+	"testing/quick"
+
+	"github.com/FurmanovD/go-kit/randomstring"
+)
+
+func TestMySqlConnectionString(t *testing.T) {
```

Comment:
The cyclomatic complexity of this function is higher that it should be. Using if conditions in tests is a code smell

Reply:
refactored